server/views/pages/activities/unlock-list/planned-events.njk

```diff
@@ -147,6 +147,34 @@
                     ]
                 }) }}
 
+                {% if activityCategories %}
```

Comment:
Are the categories in the right place [according to figma](https://www.figma.com/design/H43qocfvR2CWcnnKm3T43j/Activities?node-id=6949-6506&node-type=frame&t=LZuOwe6XCcM3WkFS-0)?